internal/pkg/cli/env_deploy.go

```diff
@@ -37,6 +42,24 @@ type deployEnvOpts struct {
 	unmarshalManifest func(in []byte) (*manifest.Environment, error)
 }
 
+// Validate returns an error for any invalid optional flags.
+func (o *deployEnvOpts) Validate() error {
+	return nil
+}
+
+// Ask prompts for and validates any required flags.
+func (o *deployEnvOpts) Ask() error {
+	if o.appName != "" {
```

Comment:
```suggestion
	if o.appName == "" {
```
to simplify the `if else`

Reply:
This was intended to re-enforce the "validateOrAsk" mindset.